### PR TITLE
Minor edits of Time based releases policy

### PR DESCRIPTION
**Pull request by t8m**
This is to improve the rendering of the policy at www.openssl.org

### CODE REVIEW

**Review by arapov — CHANGES_REQUESTED** [review]

File: policies/release-policy.md
Comment (t8m) [reply policies/release-policy.md:65]: It is intentional as the rendering on  https://www.openssl.org/policies/general/release-policy.html is confusing - it duplicates the footnote.

Comment (arapov) [reply policies/release-policy.md:65]: I see, Ok then.

**Review by arapov — APPROVED** [review]

**Comment by t8m** [comment]
One more OMC member approval needed before merging @iamamoose @paulidale @t-j-h ?

**Comment by t-j-h** [comment]
Approved with the minor changes.

**Review by t-j-h — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
@mattcaswell can you please push this? I do not have write access to the primary repo for general-policies.

**Comment by paulidale** [comment]
Pushed.

**Pull request closed without merging**
